PureScript 0.13.0 doesn't build with cabal

Running `cabal new-install purescript` yields various errors of the sort

```
      in the definition of data constructor ‘HappyWrap180’
     |
1130 | newtype HappyWrap180 = HappyWrap180 ([(SourceToken, _)])
     |                                                     ^

dist/build/Language/PureScript/CST/Parser.hs:1137:53: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap181’
     |
1137 | newtype HappyWrap181 = HappyWrap181 ([(SourceToken, _)])
     |                                                     ^

dist/build/Language/PureScript/CST/Parser.hs:1144:50: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap182’
     |
1144 | newtype HappyWrap182 = HappyWrap182 (NE.NonEmpty _)
     |                                                  ^

dist/build/Language/PureScript/CST/Parser.hs:1151:50: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap183’
     |
1151 | newtype HappyWrap183 = HappyWrap183 (NE.NonEmpty _)
     |                                                  ^

dist/build/Language/PureScript/CST/Parser.hs:1158:50: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap184’
     |
1158 | newtype HappyWrap184 = HappyWrap184 (NE.NonEmpty _)
     |                                                  ^

dist/build/Language/PureScript/CST/Parser.hs:1165:53: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap185’
     |
1165 | newtype HappyWrap185 = HappyWrap185 ([(SourceToken, _)])
     |                                                     ^

dist/build/Language/PureScript/CST/Parser.hs:1172:53: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap186’
     |
1172 | newtype HappyWrap186 = HappyWrap186 ([(SourceToken, _)])
     |                                                     ^

dist/build/Language/PureScript/CST/Parser.hs:1179:53: error:
    Wildcard ‘_’ not allowed
      in the definition of data constructor ‘HappyWrap187’
     |
1179 | newtype HappyWrap187 = HappyWrap187 ([(SourceToken, _)])
     |                                                     ^
```

I believe the package bounds should also be restricted, see https://matrix.hackage.haskell.org/#/package/purescript

--- a/.gitignore
+++ b/.gitignore
@@ -2,6 +2,8 @@ dist
 cabal-dev
 .cabal-sandbox
 cabal.sandbox.config
+dist-newstyle/
+cabal.project.local*
 *.o
 *.hi
 *.chi

--- a/package.yaml
+++ b/package.yaml
@@ -89,6 +89,8 @@ dependencies:
   - unordered-containers
   - utf8-string >=1 && <2
   - vector
+build-tools:
+  - happy ==1.19.9
 
 library:
   source-dirs: src

